ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.13-lp-interop.yaml

```
@@ -0,0 +1,39 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: docker
+  to: fuse-online-test-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: fuse-online-interop-aws
+  cron: 0 6 * * 1
```

Please add [gangway](https://cspi-qe.gitbook.io/ocp-ci-onboarding/tutorials/triggering_guide#how-to-add-gangway-api-triggering-to-a-scenario) functionality to support manually trigger for this scenario 
```suggestion
  cron: 0 6 * * 1
  remote_api: true
```